nx-dev/ui-markdoc/src/lib/tags/terminal-video.component.tsx

```diff
@@ -0,0 +1,29 @@
+import { VideoLoop } from './video-loop.component';
+import { Schema } from '@markdoc/markdoc';
+import { TerminalShellWrapper } from '../nodes/fences/terminal-shell.component';
+
+export const terminalVideo: Schema = {
+  render: 'TerminalVideo',
+  attributes: {
```

Comment:
`isMessageBelow` seems to be missing

Reply:
don't think I need it here, so I'll just pass it ahead to the wrapper where it is needed